moto/cloudformation/utils.py

```diff
@@ -10,6 +10,11 @@ def generate_stack_id(stack_name):
     return "arn:aws:cloudformation:us-east-1:123456789:stack/{0}/{1}".format(stack_name, random_id)
 
 
+def generate_changeset_id(changeset_name):
+    random_id = uuid.uuid4()
+    return 'arn:aws:cloudformation:us-east-1:123456789:changeSet/{0}/{1}'.format(changeset_name, random_id)
```

Comment:
Cloudwatch should have region specific ARNs right?

Reply:
Yes - I was again just copying convention used in `generate_stack_id` above.  Trying to be predictable.

Reply:
Fair enough, if your up for fixing all of them to use regions, by all means ;-)